### DISCUSSION TITLE
Microgame Template

### DISCUSSION THREAD

**Discussion by iznaut** [NEW-CYLANDIA/little-warioware#8] [General]
should contributors be expected to clone this repo or can we make a more barebones project for microgame development?

**Comment by TaylorAnderson** [comment]
feels like itd be nice to have two projects, so that people who might be overwhelmed by the systems work that goes into the main repo can just like go off into their own corner and make a microgame...the problem is i got no clue how to like, pull in other godot projects, other than to like have the microgame repo not be its own godot project, but be like something you would clone INTO an existing godot project?

**Comment by iznaut** [comment]
my thinking is that the template would be its own project and folks could submit the whole thing, but we'd do some processing to pull out _only_ the non-standard files so they can be included in the build (which kinda feeds into #2, which is a rabbit hole that we never quite solved with little mario 🥲)
